flink/src/main/java/org/apache/iceberg/flink/FlinkTypeVisitor.java

```diff
@@ -30,7 +30,7 @@
 import org.apache.flink.table.types.logical.YearMonthIntervalType;
 import org.apache.flink.table.types.logical.ZonedTimestampType;
 
-abstract class FlinkTypeVisitor<T> implements LogicalTypeVisitor<T> {
+public abstract class FlinkTypeVisitor<T> implements LogicalTypeVisitor<T> {
```

Comment:
Does this need to be public? The only reference to `FlinkTypeVisitor` that I see in this PR is here, so I'm not sure why this is needed.

Reply:
No need, I used to think the reading and writing will rely on `FlinkTypeVisitor`.